### PR TITLE
Add Github badge

### LINKED ISSUE #11: Add Github badge
We want to have a Github badge at the top right corner, so people know that they can contribute to the list.

### PR DESCRIPTION
**Pull request by ghost**
Added a GitHub commit activity badge at the top right corner

Fixes #11

### CODE REVIEW

**Comment by FrederikBolding** [comment]
@mcaibad2 Can we use this one instead? https://github.com/skratchdot/react-github-corner

EDIT: My bad, added the correct link.

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Use https://github.com/skratchdot/react-github-corner for the badge instead!

**Comment by mashiromashi** [comment]
Why not just add with a tag with some css ? Do we really need to install a library for this ?

**Comment by FrederikBolding** [comment]
@mashiromashi The library just wraps a svg in a component, making it prettier in our source code 😄

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Please position it in the top right of the page instead of the header 😄

**Comment by mashiromashi** [comment]
I see. Thanks for clarification.

**Comment by deamme** [comment]
You should keep index.js as it is.

File: src/components/Layout/index.js [hunk 1]
```diff
@@ -12,7 +14,9 @@ export default ({ children }) => (
         href="https://cdn.jsdelivr.net/npm/inter-ui@3.10.0/inter.min.css"
       />
     </Helmet>
-
+    <div className="githubCorner">
```
Comment (deamme) [inline src/components/Layout/index.js]: I don't think you need to wrap it in a div

File: src/pages/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react"
 
-import Layout from "../components/Layout"
 import EntryList from "../components/EntryList"
+import Layout from "../components/Layout"
```
Comment (deamme) [inline src/pages/index.js]: Can you leave this file be?

**Pull request merged**